src/index.js

```diff
@@ -28,12 +30,75 @@ router.get('/', (req, res) => {
     Number of customers out cannot be greater than 'curr'.
     Number of customer in will vary with time of day.
 */
-router.get('/customer-io', (req, res) => {
+router.get("/customer-io", (req, res) => {
+  var tooFull = false;
+  let UTCOffset;
+  var currHour = new Date().getUTCHours();
+
+  if (req.query.max == null || req.query.curr == null) {
+    res.status(400).send({});
+    return;
+  }
+
+  if (req.query.timezone != null) {
+    let timezone = req.query.timezone;
+    if (timezones[timezone] == null) {
+      res.status(400).send({}); // Unssuported timezone
```

Comment:
Are there ways to send back more meaningful error messages?

I was just thinking that, if I were the backend, I would find it helpful to distinguish between having the query.max or query.curr be null, versus being in an unsupported timezone.

Reply:
This should be fixed now.